utils/src/api_schema/api_schema.json

```diff
@@ -113,13 +113,14 @@
     "blockResults": {
       "type": "object",
       "properties": {
-        "list":     { "type": "array" },
-        "average":  { "type": "string" },
-        "count":    { "type": "string" },
-        "sum":      { "type": "string" },
-        "limit":    { "$ref": "#/definitions/positiveInt" },
-        "sort":     { "type": "string" },
-        "blob":     { "type": "boolean" }
+        "list":       { "type": "array" },
+        "average":    { "type": "string" },
+        "count":      { "type": "string" },
+        "sum":        { "type": "string" },
+        "limit":      { "$ref": "#/definitions/positiveInt" },
+        "sort":       { "type": "string" },
+        "descending": { "type": "boolean" },
```

Comment:
Is there no way to make this part of the sort command? Descend keyword by itself doesn't make sense. It is useful only in the sort context.

Reply:
yep, I was divided about this as well. Adding a "sort block" that has a prop for sorting and the option of using order = "ascending" or order = "descending" was another option. We an go with that, I think I like it more, but it was 50/50.

Reply:
Will it still be possible to have a default so you can only specify the key ?

Reply:
Ok. 
The Options are:

1)  (breaks backward compatibility)
"sort": { 
    "key": "property_used_for_sorting", 
    "order": "ascending" // or "descending", with the default to one of them. 
}

2) As it is implemented, just adds the order as an extra element of the results block, that will only be used if "sort" is also specified. 

3) We can change "descending" = [True, False] to "order" = ["ascending", "descending"]. I like this option more, which does not break compatibility, it just adds more flexibility.

Reply:
Is there no way to support either a value or a block, as:
    "sort" : "\<key\>"
and
    "sort" : { "key" : "\<key\>", "order" : "descending" }

If not, I'm inclined to go with the backward compatible option, with the addition
     "sort-order" = ["ascending","descending"]